build.gradle

```diff
@@ -172,6 +172,7 @@ dependencies {
 
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '1.0.0'
```

Comment:
There is [such](https://github.com/hmcts/bulk-scan-processor/pull/60/files#diff-c197962302397baf3a4cc36463dce5eaR179) dependency already